usr/src/uts/common/brand/lx/syscall/lx_socket.c

```diff
@@ -2825,7 +2826,7 @@ static const lx_sockopt_map_t ltos_tcp_sockopts[LX_TCP_NOTSENT_LOWAT + 1] = {
 	{ TCP_LINGER2, sizeof (int) },		/* TCP_LINGER2		*/
 	{ OPTNOTSUP, 0 },			/* TCP_DEFER_ACCEPT - in code */
 	{ OPTNOTSUP, 0 },			/* TCP_WINDOW_CLAMP - in code */
-	{ OPTNOTSUP, 0 },			/* TCP_INFO		*/
+	{ TCP_INFO, sizeof (tcp_info_t) },	/* TCP_INFO		*/
```

Comment:
I don't see tcp_info_t defined anywhere, and if it's LX-only, it should be `lx_tcp_info_t`.

Reply:
The change is not just for LX is for OS as well.